packages/utils/package.json

```diff
@@ -74,7 +74,6 @@
     "@rollup/plugin-json": "^6.0.0",
     "@rollup/plugin-node-resolve": "^15.1.0",
     "@waku/build-utils": "*",
-    "@waku/interfaces": "0.0.17",
```

Comment:
It's not strictly important but types usually are `devDependencies`.

Reply:
Apart when we use the protocol enum for this package

Reply:
or `types` are part of functionality (it semantically makes more sense)